tests/playwright/src/utility/operations.ts

```diff
@@ -144,6 +147,17 @@ export async function deletePod(page: Page, name: string): Promise<void> {
   }
 }
 
+export async function createPod(podsPage: PodsPage, podAppName: string): Promise<void> {
+  await playExpect(podsPage.heading).toBeVisible();
+
+  const playYamlPage = await podsPage.openPlayKubeYaml();
+  await playExpect(playYamlPage.heading).toBeVisible();
+
+  const yamlFilePath = path.resolve(__dirname, '..', '..', 'resources', `${podAppName}.yaml`);
+  podsPage = await playYamlPage.playYaml(yamlFilePath);
+  await playExpect(podsPage.heading).toBeVisible();
+}
+
```

Comment:
Playing yaml file does not necessarily mean that a pod is created, that is just the consequence of the test that this runs, but it does not make sense for the method to be named as such, since playing a yaml could just as easily not create a pod.

Also it's not a good idea to abstract the path to the yaml file this way at all, basically you are saying that only a yaml file in the resources folder can ever be used, what if we need to create a test with a yaml file in a different location?

Also I don't really understand why this method even needs to exist, the playYaml() method already exists, refactoring the entire test into one bigger method just creates needless abstractisation and makes it much harder to actually use.

If you really want to encapsulate the entire functionality, this should be a method exposed from the PodsPage POM classbut which receives the entire path to the yaml file, not just the name and also should be renamed.